app/views/items/index.html.erb

```diff
@@ -21,5 +21,49 @@
         <p>Inactive</p>
       <% end %>
     </section>
-    <% end %>
+  <% end %>
+</section>
+<section class="grid-container">
+  <article class="popular-items">
+  <h1>Most Popular</h1>
+  <% @items.most_popular(5).each do |item| %>
+    <section class = "grid-item" id= 'item-<%=item.id%>'>
+      <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
+      <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
+      <img src= <%= item.image %>>
+      <p> <%= item.description unless @merchant%> </p>
```

Comment:
US 17 requires us to have the items images on the index page be links to the show page, should we add that here?